collection-scripts/gather_priority_and_fairness

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="/must-gather"
+NAMESPACE_PATH="namespaces/openshift-kube-apiserver"
+POD_PATH="pods/kube-apiserver/kube-apiserver"
+APF_PATH="${BASE_COLLECTION_PATH}/${NAMESPACE_PATH}/${POD_PATH}/api_priority_and_fairness"
```

Comment:
thinking more about the location, could we place to 

`openshift-kube-apiserver/pods/kube-apiserver-ip-10-0-145-102.us-east-2.compute.internal/kube-apiserverkube-apiserver/api_priority_and_fairness`

replacing `kube-apiserver-ip-10-0-145-102.us-east-2.compute.internal` for each server ?